packages/react-devtools/app.html

```diff
@@ -116,8 +132,14 @@
           <div class="box-header">React DOM</div>
           <div class="box-content">
             Add one of the following:
```

Comment:
Could we just change this to:

> Add one of the following (click to copy):

and then automatically copy the text for them? (without adding new buttons)